{{cookiecutter.slug}}/frontend.angular/Dockerfile-postgenerate

```diff
@@ -0,0 +1,51 @@
+FROM node:18 AS bootstrap-1
+
+WORKDIR /usr/src/app
+RUN yarn init -yp && \
+    yarn add "@angular/cli@17" && \
```

Comment:
Better to install a newer version of Angular, once we're at it, compatible with a newer node version.